### ISSUE TITLE
broken link for INT Historical Word List

### ISSUE THREAD

**Issue by kreetrapper** [clarin-eric/clarin-resource-families#15]
The handle no longer resolves. This should be fixed by INT.

**Comment by VincentCCL** [comment]
The handle http://hdl.handle.net/10032/tm-a2-a6 resolves nicely to https://taalmaterialen.ivdnt.org/download/tstc-int-historische-woordenlijst/
The publication link is not correct.

**Comment by kreetrapper** [comment]
Indeed. The handle resolves. I assume there was a temporary problem, when I tried this. 

The link to the publication on the INT website does not seem to work, though. Can you look into fixing that link and provide the reference, so we can properly add it to the CRF item?

**Comment by VincentCCL** [comment]
The paper seems to be https://doi.org/10.1117/12.2008423, the year of the ref should change to 2013.
